shanoir-ng-import/src/main/java/org/shanoir/ng/importer/ImporterApi.java

```diff
@@ -90,6 +90,23 @@ ResponseEntity<Void> uploadFile(
     @PreAuthorize("hasRole('ADMIN') or (hasAnyRole('EXPERT', 'USER') and @importSecurityService.hasRightOnOneStudy('CAN_IMPORT'))")
     ResponseEntity<ImportJob> uploadDicomZipFile(@ApiParam(value = "file detail") @RequestPart("file") MultipartFile dicomZipFile) throws RestServiceException;
     
+    @ApiOperation(value = "Upload multiple examinations DICOM .zip file", notes = "Upload DICOM .zip file", response = Void.class, tags={ "Upload one DICOM .zip file", })
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "success returns file path", response = Void.class),
+        @ApiResponse(code = 400, message = "Invalid input / Bad Request", response = Void.class),
+        @ApiResponse(code = 409, message = "Already exists - conflict", response = Void.class),
+        @ApiResponse(code = 200, message = "Unexpected Error", response = Error.class) })
+    @PostMapping(value = "/upload_multiple_dicom/study/{studyId}/studyName/{studyName}/studyCard/{studyCardId}/center/{centerId}/converter/{converterId}/",
```

Comment:
Dear Jean-Come, thx for the PR. Is it really necessary to use studyId+studyName in the URL?
MS Datasets can resolve studyId to name? Is not there a risk, that the studyName might contain
special chars, that are complicated to express in URLs?

Reply:
My point of view was that all these informations are available from the front and we can directly send them, instead of making multiples "risky" (rabbitMQ + SQL requests) calls in the back.
For the studyName, it's true, that's why I wanted to send it in an 'importJob' object in JSON, but I could not find how to send a JSON object along with a File -> We can imagine infering it from the database, I'll add a StudyRepository in import MS

Reply:
Actually we don't have Study table in import database, so we have to load it using rabbitMQ.

Reply:
sorry, I agree, I made a mistake and mixed up between ms datasets and ms import
when you create the subject, ms studies knows the studyName, we could let manage
the subjectName creation within ms studies?

Reply:
We need the studyname and the subject name in the importJob in case of failure for the failure mail, so we can't ignore it in import side. (otherwise, we can just put the study id but it's not really clear for end users)